lib/goodcity/organisation_populator.rb

```diff
@@ -22,7 +22,7 @@ def initialize
     end
 
 
-    def populate_organisation_db
+    def run
```

Comment:
I was wondering what name to use, so ended up with that. Anyway run sounds better.